jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml

```diff
@@ -7,7 +7,7 @@
             days-to-keep: 7
     provider-env: |
         export KUBERNETES_PROVIDER="gce"
-        export E2E_MIN_STARTUP_PODS="1"
+        export E2E_MIN_STARTUP_PODS="8"
```

Comment:
Im afraid that will not work, because we are not starting all addons in kubemark
@gmarek